torch/testing/_internal/common_methods_invocations.py

```diff
@@ -588,22 +588,48 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad, **kw
 
     return inputs
 
+def sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+    tests_list = [
+        ((S, M), (S, S), (S, M)),
```

Comment:
How long do these tests run? We could change these to smaller values.

Reply:
I haven't timed them, but I can certainly change them to smaller values.
How about replacing S &M with 2 & 3, respectively?

Reply:
SG